### DISCUSSION TITLE
Want to enable verse text selection

### DISCUSSION THREAD

**Discussion by oddpaw** [ezra-bible-app/ezra-bible-app#1092] [Ideas]
When reading the Bible, I often come across unfamiliar words. It would be really helpful if I could select these words and copy them for quick lookup in various dictionaries.

## Some backgrounds

If the Bible is in English, typing out the unfamiliar words on a keyboard is a bit inconvenient but manageable. However, when the Bible is in other languages, especially Japanese, Korean, or Chinese, it becomes much harder to identify the characters, making it difficult to know how to type them. Therefore, being able to copy the text directly from the Bible would be incredibly useful.

Additionally, it would be ideal if only the selected text could be copied, without including the book and chapter references.

Thanks

**Comment by tobias-klein** [comment]
@oddpaw Thank you for making this suggestion for an improvement. I see where you are coming from. I will investigate the technical possibilities to see whether we could implement verse text selection and then get back.

**Reply by oddpaw** [reply]
Thanks you

**Reply by tobias-klein** [reply]
@oddpaw I have created this issue: #1104
This issue will be resolved in the next release.

**Reply by oddpaw** [reply]
Wow, that was super quick! Thanks

**Comment by tobias-klein** [comment]
@oddpaw This enhancement is now available as part of the [new release 1.15](https://github.com/ezra-bible-app/ezra-bible-app/releases/tag/1.15.0)!
